api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -375,7 +375,6 @@ public void getAge_shouldGetAgeAfterDeath() {
 		deathDate.set(2000, Calendar.JUNE, 3);
 		Person person = new Person();
 		person.setBirthdate(birthdate.getTime());
-		person.setDead(true);
```

Comment:
Because it is a public method and hence can be called by the API consumer. So what would happen when one calls person.setDead(true)?